pkgs/applications/misc/findex/default.nix

```
@@ -0,0 +1,32 @@
+{ lib, fetchFromGitHub, rustPlatform, pkg-config, keybinder3, gtk3 }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "findex";
+  version = "v0.7.1";
+
+  src = fetchFromGitHub {
+    owner = "mdgaziur";
+    repo = pname;
+    rev = version;
+    hash = "sha256-KaT6lEbrUelv/f9bIBW4bSCuExFu4b7XI7hcrO4mD0M=";
+  };
+
+  cargoHash = "sha256-tGn+G9TvQWF8J1BGKYqn7JhYK2hf4HFCNdV3HKYzvJA=";
+
+  nativeBuildInputs = [
+    pkg-config
+  ];
+
+  buildInputs = [
+    gtk3
+    keybinder3
+  ];
+
+  meta = with lib; {
+    description = "A highly customizable application finder written in Rust and uses Gtk3";
+    homepage = "https://github.com/mdgaziur/findex";
+    license = licenses.gpl3Only;
+    platforms = ["x86_64-linux" "i686-linux" "aarch64-linux"];
```

```suggestion
    platforms = platforms.linux;
```

Or simple for our usecases